include/engine/routing_algorithms/routing_base.hpp

```diff
@@ -484,6 +484,69 @@ template <class DataFacadeT, class Derived> class BasicRoutingInterface
         unpacked_path.emplace_back(t);
     }
 
+    /**
+     * A duplicate of the above `UnpackEdge` function, but returning full EdgeData
+     * objects for the unpacked path. Used in the tile plugin to find outgoing
+     * edges from a given turn.
+     */
+
```

Comment:
To further clarify. NodeBasedDynamic graph offers findSmallestEdge (https://github.com/Project-OSRM/osrm-backend/blob/master/include/util/dynamic_graph.hpp#L254) which adds exactly the functionality used here in the modified `unpack`.

So a normal `unpack` and following use of `findSmallestEdge` should result in just the same output, which means the functionality here is technically already implemented.